src/dotnet/Notification.Service/Notifications.Backend.cs

```diff
@@ -163,15 +162,15 @@ public virtual async Task<ImmutableArray<Symbol>> ListSubscriberIds(string chatI
 
         }
 
-        async Task<string> AuthorNameResolver(string authorId)
+        async Task<string> AuthorNameResolver(string authorId, CancellationToken ct)
```

Comment:
for oarameters of nested methods we use `1` suffix - `cancellationToken1`

Reply:
done